src/backend/marsha/core/tests/test_views_lti_video.py

```diff
@@ -102,7 +102,10 @@ def test_views_lti_video_post_instructor(self, mock_get_consumer_site, mock_veri
         self.assertEqual(context.get("state"), "success")
```

Comment:
Commit messages:

✨(backend/frontend) add support to serv**e** static **images**
Because frontend needs to access static images (png, jpeg, etc) and to be able to
use CDN **, the** backend will expose static endpoints in app data.
Frontend will be able to use those endpoints to fetch **resources**.